src/pages/components/select/usage.mdx

```diff
@@ -345,7 +348,7 @@ The select has multiple interactive states for both default and inline variants:
 
 <Tab label="Default">
 
-![Select interactive states](images/select-states-fixed.png)
+![Select interactive states](images/select-states-default.png)
```

Comment:
Read-only state (fixed): You can take out the helper text here since you don’t show it for the other states.